doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -231,7 +229,29 @@ It defines the end-end Certification Process required for certifying the given V
 using this platform (providing above requirements), producing the test results to
 certification portal , review and granting the badge to this VNF.
 
-[More details](https://wiki.lfnetworking.org/display/LN/VNF+Certification+Lifecycle)
+The complete process is detailed below:
+
+1. *VNF Vendors* submit the VNF into OVP Lab for certification testing_guide
+
+2. Using VNF Test Platform, VNF will be tested with all required test cases Provided
+by *Test authors* by using the appropriate MANO and NFVI provided by Vendors. while
+testing, required profile to be considered. This whole setup is owned by 3rd-Party
+ *OVP Lab provider*
+
+ ![](media/f3b0c214bc58c44406fd5b801d3dfc89.png)
+
+3. Once testing is completed successfully, test results will be submitted to the OVP
+ portal for community review.  When submitting the application to the portal, the vendor will also provide a additional information, including the product name, documentation links, primary company contact information, etc.
+
+4. *LFN CVC* reviewers will review the results and will approve or reject it.
+
+5.  If reviewer rejected it, then step 2 and 3 will be ran again to address the
+review comments. Otherwise Once reviewer approved it, corresponding VNF will be
+published into OVP VNF Portal with OVP badge.

```

Comment:
For an approved VNF, will there be a situation that it works with X version of NFVI and MANO and breaks with future certified releases say Y? If yes, it means that NFVI version X lost the interoperability going to version Y. To handle such situations, OVP VNF portal should list the version details of used NFVI and MANO for that VNF.

Reply:
@deebhatia What version would recommends, version of open stack, version of CNTT release, version of the "installer"?

Reply:
OpenStack. CNTT release and installer will internally refer a specific OpenStack version.

Reply:
I've added a ticket to the dovetail Jira to track adding the OpenStack version to the Dovetail portal for VNF tests.